change/@fluentui-azure-themes-09726f52-8b4c-4029-b2bc-0c71b8f8660e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30420&commitId=47c3077b1d4c4768c0542f31e25d55c9eb05060a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Image</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.Fit- none, image smaller.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>